crates/aptos/src/test/mod.rs

```diff
@@ -898,6 +898,7 @@ impl CliTestFramework {
             package,
             output_dir: Some(output_dir),
             print_metadata: false,
+            bytecode: true,
```

Comment:
Would be good to add a decompile test later (but of course that means we have to download the decompiler, so we can skip for now)

Reply:
It appears there are a number of commands we do not have e2e tests for: download (or is it tested indirectly), list, disassemble, and now decompile.